CLI/commands/common/common_functions.js

```diff
@@ -25,6 +26,38 @@ async function checkPermission(contractName, functionName, contractRegistry) {
   }
 };
 
+function getFinalOptions(options) {
```

Comment:
It should check if 'options' is undefined. This way we don't need to send '{}' on every sendTransaction and it will be totally optional.

Reply:
I understand, but the idea behind the explicit '{}' is remember that you could overwrite any params regarding tx.